README.md

```diff
@@ -5,7 +5,26 @@
 Kitodo.Publication is free software, an extension for [TYPO3](https://typo3.org/) and part of the [Kitodo Digital Library Suite](https://en.wikipedia.org/wiki/Kitodo).
 It implements the user and administrator interfaces for a [document and publication server](https://en.wikipedia.org/wiki/Institutional_repository).
 
+## DDEV Development Environment
+
+This extension provides a [DDEV](https://ddev.readthedocs.io/en/stable/) TYPO3 environment. On initial checkout TYPO3 needs to be installed and configured via composer. Prior to running TYPO3 import inital database dump.
+
+### Start and Configuration
+1. `ddev start` to start all containers
+2. `ddev composer install` to install TYPO3 and all extensions
+3. `ddev import-db -f db.sql.gz` to import the prepared database
+4. `ddev launch typo3` to go to the backoffice login page
+
```

Comment:
The explanation before the steps is confusing and makes more sense at the actual steps. Furthermore, I would link to the DDEV site and the documentation separately. :)

```suggestion
This extension provides a TYPO3 environment powered by [DDEV](https://www.ddev.com). For more information check out the [DDEV documentation](https://ddev.readthedocs.io/en/stable/).

### Start and Configuration
1. `ddev start` to start all containers
2. `ddev composer install` to install TYPO3 and all extensions
3. `ddev import-db -f db.sql.gz` to import the prepared database
4. `ddev launch typo3` to go to the backoffice login page

Steps 1–3 are mandatory after the initial checkout to set up the virtual environment.

```